ngraph/core/include/ngraph/op/op.hpp

```diff
@@ -36,3 +37,10 @@ namespace ngraph
         };
     }
 }
+
+#define NGRAPH_OP_CREATE_NODE(OP)                                                                  \
```

Comment:
Why do we need to have this define?

Reply:
For consistency. You have to always call `make_shared` because `Node` inherit from `enable_share_from_this` and only `Constant` has `create` method. This macro will introduce `create` method for each Op.